_posts/2015-09-06-beautiful-jekyll.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Beautiful Jekyll
+date: 2015-09-06 00:00:00
+homepage: https://github.com/daattali/beautiful-jekyll
+download: https://github.com/daattali/beautiful-jekyll/archive/master.zip
+demo: http://deanattali.com/beautiful-jekyll/
+author: Dean Attali
+thumbnail: beautiful-jekyll.png
+license: MIT
```

Comment:
Hello! To assure the similar formatting and style for all themes please change the `MIT` to `MIT License`.